### PR TITLE
getDatabase.php: update the s3cmd config and hosts for Poznań devboxes

### PR DESCRIPTION
**Pull request by macbre**
See https://github.com/Wikia/config/pull/1194

And update the s3 buckets used to get the database dumps - https://wikia-inc.atlassian.net/browse/OPS-6676 / https://github.com/Wikia/chef-repo/pull/5945

@drozdo / @wladekb / @michalroszka / @owend

### CODE REVIEW

**Comment by drozdo** [comment]
LGTM

**Comment by ghost** [comment]
LOOKS_GOOD-to-me

File: maintenance/wikia/getDatabase.php [hunk 1]
```diff
@@ -75,14 +77,10 @@
...
-		 * 1->A, 2->B
+		 * 1->a, 2->b
 		 * chr(65)=='A' */
 		$clusterLetter = chr( 64 + $clusterNumberParam );
-		$databaseDirectory = "database_{$clusterLetter}";
-		if ( $clusterNumberParam >= 6 ) {
-			// Way of combining s3_bucket name changed from cluster 6
-			$databaseDirectory = 'database-' . strtolower( $clusterLetter );
-		}
+		$databaseDirectory = 'database-' . strtolower( $clusterLetter );
```
Comment (owend) [inline maintenance/wikia/getDatabase.php:83]: The directory names were different for different clusters (some use - and some use _) but it looks like this works now.  Did we change this on S3?

Comment (macbre) [inline maintenance/wikia/getDatabase.php:83]: Yes, we did that few weeks ago. @drozdo will know the details.

**Pull request merged**